pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -229,5 +228,7 @@ stdenv.mkDerivation rec {
     homepage = "https://kotatogram.github.io";
     changelog = "https://github.com/kotatogram/kotatogram-desktop/releases/tag/k{version}";
     maintainers = with maintainers; [ ilya-fedin ];
+    # never built on aarch64-darwin since first introduction in nixpkgs
+    broken = (stdenv.isDarwin && stdenv.isAarch64) || (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
Not sure if we would want to use stdenv.isAarch64 here only or not

Reply:
My gut feeling says that's not a good idea, but it can be considered.